tests/__init__.py

```diff
@@ -16,7 +16,7 @@ class Species(str, Enum):
 
 class Pet(BaseModel):
     name: str
-    species: Species
+    species: Species = Field(default=Species.MONKEY)
```

Comment:
do you not have to give it an ENUM type in the Field() def? unsure

Reply:
not at all, but i wanted to check that some lines got covered - and this was required.